docs/other/model-in-memory.mdx

```diff
@@ -0,0 +1,55 @@
+---
+id: model_in_memory
+title: Creating AMF model in memory
```

Comment:
Creating an AMF model in memory

Reply:
Also @arielmirra I'm sorry I haven't noticed this until now, but I think we've been going back and forth on title and heading capitalization. I think we should standardize and use title case or sentence case for titles and headings. 

That means:

Creating a Model in Memory Or Creating a model in memory

and similarly throughout. If it's easier, I can create a separate ticket to PR that

Reply:
You're right @IsaacEldridge! if you can take a look into it in a different PR that would be nice, if you need me to create a ticket let me know 👍🏽